webapp/app/lang/en.json

```diff
@@ -0,0 +1,30 @@
+{
+  "button": {
+    "add": "Add",
+    "cancel": "Cancel",
+    "delete": "Delete",
+    "edit": "Edit",
+    "save": "Save",
+    "sign_in": "Sign in"
+  },
+  "text": {
+    "action_user_form": "%{action} User",
+    "back_to_top": "Back to top",
+    "delete_user": "Are you sure to delete User %{name}",
+    "footer": "Directly starter app in React Redux. See this project @",
+    "github": "GitHub",
+    "go_back": "Go back to %{page}",
+    "header": "Directly starter app",
+    "hint": "Hint:",
+    "hint_username_password": "username/password",
+    "email": "Email",
+    "name": "Name",
+    "password": "Password",
+    "page_not_found": "Page not found",
+    "phone_number": "Phone Number",
+    "please_sign_in": "Please sign in",
+    "skype_id": "Skype ID",
+    "users_list": "Users List",
+    "username_or_email": "Username or email address"
+  }
+}
```

Comment:
What do you think about organizing keys by module EG:

```
homePage: {
  "users:_list": "...",
},
actionButtons: {
  ""
},
usersForm: {
  ...
"phone_number": "Phone Number",
  ...
}
```

Reply:
Could be a good approach, but in this case, you can repeat keys with this approach. For example username, email and phone are repeated for the table and modal. Depends on the project, the best configuration to do.
![i18nkeys](https://user-images.githubusercontent.com/6341150/43478489-65eef13e-94d4-11e8-908a-4438d896473f.png)

Reply:
ok I agree